src/Lightning.ts

```diff
@@ -305,7 +326,7 @@ export const LightningMixin = (superclass) =>
           if (isMyNode({ pubkey: destination }) || destination === "") {
             const lightningLoggerOnUs = lightningLogger.child({ onUs: true, fee: 0 })
 
-            if (await this.user.limitHit({ on_us: true, amount: tokens })) {
+            if ((await this.user.remainingOnUsLimit) < tokens) {
```

Comment:
I would generally avoid doing an await + comparison + if in the same line. you can extract   `remainingOnUsLimit` and set it as a const in the line above for more clarify?

Reply:
Done